src/rss/filing_tracker.py

```
@@ -0,0 +1,290 @@
+                "filing_id": filing_id,
+                "case_number": filing.get('case_number', 'Unknown'),
+                "debtor_name": filing.get('debtor_name', 'Unknown'),
+                "court": filing.get('court', 'Unknown'),
+                "court_code": filing.get('court_code', 'Unknown'),
+                "chapter": chapter,
+                "filing_type": filing.get('filing_type').value if hasattr(filing.get('filing_type'), 'value') else filing.get('filing_type', 'unknown'),
+                "title": filing.get('title', ''),
+                "description": filing.get('description', ''),
+                "link": filing.get('link', ''),
+                "published_date": filing.get('published', ''),
+                "detected_at": datetime.utcnow().isoformat(),
```

Ruff (0.11.9)

`datetime.datetime.utcnow()` used

(DTZ003)